redisson/src/main/java/org/redisson/RedissonReactive.java

```diff
@@ -389,8 +389,10 @@ public boolean isShuttingDown() {
         return connectionManager.isShuttingDown();
     }
 
-    protected void enableRedissonReferenceSupport() {
+    protected void enableRedissonReferenceSupport(Config config) {
         this.commandExecutor.enableRedissonReferenceSupport(this);
+        Codec codec = config.getCodec();
```

Comment:
@jackygurui Please encapsulate this logic into `CommandAsyncService.enableRedissonReferenceSupport` method to have all stuff in single place.

Reply:
Good point. It is done.